scripts/linting.sh

```diff
@@ -2,4 +2,4 @@
 set -e
 set -v
 
-python -m flake8 pantry --config=.flake8
+python -m flake8 --ignore=F403,F401 pantry --config=.flake8
```

Comment:
Needed to allow the `from .settings import *` this is a python anti-pattern, but I don't want to import everything in settings.py by name so this is a massive convenience